I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 58, in script call
    call scene11
  File "game/chapter2/scene11.rpy", line 73, in script
    jump chosen_label
ScriptError: could not find label 'chosen_label'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 58, in script call
    call scene11
  File "game/chapter2/scene11.rpy", line 73, in script
    jump chosen_label
  File "/mnt/windows/renpy-8.3.4-sdk/renpy/ast.py", line 1705, in execute
    rv = renpy.game.script.lookup(target)
  File "/mnt/windows/renpy-8.3.4-sdk/renpy/script.py", line 1103, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'chosen_label'.

Linux-6.12.7-zen1-1-zen-x86_64-with-glibc2.40 x86_64
Ren'Py 8.3.4.24120703
DreamEngine 1.0
Mon Dec 30 20:16:24 2024
